Makefile

```diff
@@ -57,7 +57,7 @@ extension: build/unload-client.js
 extension: build/pdfjs-init.js
 extension: $(addprefix build/,$(EXTENSION_SRC))
 
-build/extension.bundle.js: src/background/*.js src/background/*.ts rollup.config.mjs build/settings.json
+build/extension.bundle.js: src/background/*.ts rollup.config.mjs build/settings.json
```

Comment:
There are no longer any .js files in src/background, so remove the wildcard to avoid an error from make.